### PR TITLE
767: Fix order status for unconnected wallet

### LINKED ISSUE #767: Fix order status for unconnected wallet
Image

When opening taken order with unconnected wallet it doesn't show the correct "taken" status.

### PR DESCRIPTION
**Pull request by piersss**
Fixes #767 

Also added a loading state while the contract function is called.

https://github.com/airswap/airswap-web/assets/86911296/5db3d206-7bc3-4496-8f5f-af17c86fa49b

### CODE REVIEW

File: src/components/OrderDetailWidget/subcomponents/OrderStatusInfo/OrderStatusInfo.tsx
Comment (personnamedmike) [inline src/components/OrderDetailWidget/subcomponents/OrderStatusInfo/OrderStatusInfo.tsx:1]: There's a linting error `'LoadingSpinner' is defined but never used.` on line 6

File: src/components/OrderDetailWidget/subcomponents/OrderStatusInfo/OrderStatusInfo.tsx [hunk 1]
```diff
@@ -3,23 +3,39 @@ import { useTranslation } from "react-i18next";
 
 import { OrderStatus } from "../../../../types/orderStatus";
 import ExpiryIndicator from "../../../ExpiryIndicator/ExpiryIndicator";
-import { Button, InfoWrapper, StyledIcon } from "./OrderStatusInfo.styles";
+import LoadingSpinner from "../../../LoadingSpinner/LoadingSpinner";
```
Comment (personnamedmike) [inline src/components/OrderDetailWidget/subcomponents/OrderStatusInfo/OrderStatusInfo.tsx:6]: There's a linting error `'LoadingSpinner' is defined but never used.`

**Review by personnamedmike — APPROVED** [review]
The code seems to work, I didn't see any bugs, and the tests passed on my end. There's a linting error in OrderStatusInfo.txt, otherwise nothing jumps out

File: src/components/OrderDetailWidget/subcomponents/OrderStatusInfo/OrderStatusInfo.tsx (hunk 1 above)
Comment (piersss) [reply src/components/OrderDetailWidget/subcomponents/OrderStatusInfo/OrderStatusInfo.tsx]: Whoops, fixed!

**Pull request merged**